torch_np/_ufuncs.py

```diff
@@ -2,13 +2,14 @@
 
 import torch
 
-from . import _helpers
-from ._detail import _binary_ufuncs
+from . import _binary_ufuncs_impl, _helpers, _unary_ufuncs_impl
 from ._normalizations import ArrayLike, DTypeLike, NDArray, SubokLike, normalizer
 
-__all__ = [
+# ############# Binary ufuncs ######################
+
+__binary__ = [
```

Comment:
Not a fan of using non-standard dunder names really... Why not using a good ol' `_binary`?